### PR TITLE
 Redone PRTemplate

### PR DESCRIPTION
**Pull request by sr229**
### Description

Since the current PR Template is a bit too vague, I redone some things so people can use this template efficiently

### Status

**Finished**. Pending Merge.

### Changelog

~~- Added Additional Notes Section for Non-issues.~~ Dropped as it makes the template complex.
- Made Related Issues strictly for linking issues to be automatically resolved
- Made more clearer wording on some comments

### CODE REVIEW

**Comment by TPGPL** [comment]
i think that current PR Template is too overwhelming to new contributors

**Comment by sr229** [comment]
we can omit the proposed changes and just go with clearer wordibg if you wish @TPGPL

**Comment by sr229** [comment]
@TPGPL fixed issues and applied suggestions.

**Comment by TPGPL** [comment]
i would just leave Description and Status imo

**Comment by sr229** [comment]
@TPGPL @MegaApplePi the reason for ``Changelog`` to exist is to sum up the number of changes in subset on your PR, however, since Description seems to do the job for use, I'll just leave that be and remove everything

**Comment by sr229** [comment]
Applied your suggestions, waiting for another review

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -15,24 +16,9 @@ The status of your PR. ie WIP or Finished.
 Ex: Finished; Pending Review
```
Comment (MegaApplePi) [inline .github/PULL_REQUEST_TEMPLATE.md]: remove the comments, use a checklist (you can add a comment on how to check a box)
```
- [ ] WIP
- [ ] Pending Review
- [ ] Finished
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -15,24 +16,9 @@ The status of your PR. ie WIP or Finished.
...
-
-### Checklist <!-- Delete this if you don't have a checklist -->
-<!-- 
-To make a checklist:
-- [] Entry 1
-- [] Entry 2
--->
-
-### Related Issues <!-- Delete this if you don't have any issues related to your PR -->
+### Additional Notes <!-- Omit this if you don't have any issues related to your PR -->
```
Comment (MegaApplePi) [inline .github/PULL_REQUEST_TEMPLATE.md]: omit the comment, if there are no notes, then so be it

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -15,24 +16,9 @@ The status of your PR. ie WIP or Finished.
...
 
-<!-- If you have any issues related to this PR, link them here.
+<!--
+Any Additional notes regarding your PR goes here.
```
Comment (MegaApplePi) [inline .github/PULL_REQUEST_TEMPLATE.md]: this section is self explanatory, replace it by saying that they can close issues by using something like `resolves #589`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,4 +1,5 @@
-<!-- Please review the following before submitting a PR:
+<!-- 
+Please review the following before submitting a PR:
 osu!'s Article Styling Criteria: https://osu.ppy.sh/help/wiki/Article_Styling_Criteria
 -->
```
Comment (MegaApplePi) [inline .github/PULL_REQUEST_TEMPLATE.md]: line 10: remove the comment, the description should be self explanatory

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -1,4 +1,5 @@
-<!-- Please review the following before submitting a PR:
+<!-- 
+Please review the following before submitting a PR:
 osu!'s Article Styling Criteria: https://osu.ppy.sh/help/wiki/Article_Styling_Criteria
```
Comment (MegaApplePi) [inline .github/PULL_REQUEST_TEMPLATE.md]: if you kept the checklist section, you can remove this comment and put this in the checklist (this was what I was talking about for the checklist section)

```
### Checklist

- [ ] Reviewed [ASC](https://osu.ppy.sh/wiki/ASC)
```

Comment (sr229) [reply .github/PULL_REQUEST_TEMPLATE.md]: No real point for that kind of checklist atm, just making it sleekier and simple for now

**Comment by sr229** [comment]
Applied Suggestions, waiting for another review

**Comment by sr229** [comment]
CC @Repflez  to review for possible merge

**Comment by sr229** [comment]
This PR should've been merged already due to its finished state. Marking stale.

**Pull request merged**